packages/cloud-cognitive/src/components/Datagrid/styles/datagrid.scss

```diff
@@ -334,7 +334,7 @@
 
 .#{block-class}__displayFlex {
   position: relative;
-  display: flex;
+  display: inherit;
```

Comment:
I'm not 100% this should change. This is for providing a way to have a data grid that uses display flex. Possibly there's a different approach we could take that allows us to keep `display: flex;`

Reply:
Fixed.